### DISCUSSION TITLE
Minimize or X to icon tray

### DISCUSSION THREAD

**Discussion by kw0n** [timschneeb/GalaxyBudsClient#338] [Q&A] [answered]
Hello, will you be implementing a minimize or X to icon tray in the future so that the app does not show in the task bar if not in use?

**Comment by timschneeb** [comment, marked as answer]
This has already been implemented; however, the option is invisible in the latest release due to a bug.

You can download an update with the bugfix here: https://github.com/ThePBone/GalaxyBudsClient/actions/runs/2366081081

See here for more details about this issue: #328
